server/views/pages/recentArrivals/recentArrivalsSummary.njk

```diff
@@ -70,6 +70,17 @@
                             }
                         ]
                     }) }}
+
+                    {{ summaryCard({
+                        classes: "govuk-!-margin-bottom-6",
+                        titleText: "X-ray body scan",
+                        rows: [
+                            {
+                                key: {  text: "Total number of scans", caption: "Since 1 January " + "current year" | generateCurrentYear },
```

Comment:
Not 100% sure if generating the current year is best done via a nunjuck filter (does feel strange using a filter on the `"current year"` string above) but think also better than generating via a service method and passing to the controller? Open to thoughts on this.

Reply:
It appears to be a nice way to implement this and make it accessible across the nunjucks templates

Reply:
I think generating the year in nunjucks is nice.

it might make more sense for you to add it as a function to the nunjucks [globals](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/blob/72229af6cb1cc4656f450d98ce787c01fe80fe0a/server/utils/nunjucksSetup.ts#L95)

Then you can still call it to generate it dynamically but without having to pass a filtered string

Reply:
Thank you - added to globals as suggested in latest commit